cfme/utils/appliance/db.py

```diff
@@ -48,7 +48,11 @@ def address(self):
         if self.appliance.db_host:
             return self.appliance.db_host
         try:
-            db_addr = self.appliance.wait_for_host_address()
+            db_addr = (
+                self.appliance.wait_for_host_address()
```

Comment:
If the evmserver is down, `self_wait_for_host_address()` will take 200sec until it returns TimedOutError and `None` is assigned to `db_addr`, and since `.address` is used a lot, the test will take much much longer to execute if we have to wait for 200sec every time `.address` is called. Instead, we directly assign `None` to `db_addr` if evmserver is down.